### ISSUE TITLE
Weblog Configuration changes don't refresh the weblog list until app is restarted

### ISSUE THREAD

**Issue by nonplayer** [RickStrahl/MarkdownMonster#1137]
Hello and good day! I am trying my first attempt at using the weblog feature to upload to a wordpress blog. I am trying to add my weblog to the Web Logs tab, but it doesn't want to save. I have entered the data (Name, ID, Type, Url, Username, and Password), verified the password with the green check, and clicked Save Weblog. This has no apparent effect. The weblog does not appear in the drop-down, and if I close the dialog it is all forgotten.

I am running 3.4.4 on Windows 10, installed to "C:\Program Files\Markdown Monster" with minimal additional configuration done so far.

Thanks!

**Comment by RJS32** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by Siphompeks** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by RickStrahl** [comment]
Yes that's definitely a bug. Issue was that the bindings were not observable so although the data was updated the UI didn't reflect the list changes.

Refactored the ViewModel code to use an observable and it should now be working.

Fixed in `3.5.15..6`
